RocketChat/Rocket.Chat.ReactNative
# [FIX] Workspace input without i18n

* Strings are added for translation. Provided a default value if the translation fails.

fixes: #2620



## Proposed changes


## Issue(s)	

Some strings were not added for translation at the login screen.

## How to test or reproduce

Add translated string to respective files (such as app/i18n/locales/ar.js, app/i18n/locales/fr.js) and visit the login screen. The translated strings should be visible.

## Screenshots
<img width="314" alt="Screenshot 2020-12-04 at 7 23 22 PM" src="https://user-images.githubusercontent.com/23723464/101171753-3ba27a00-3666-11eb-991c-9c291839a595.png">

## Types of changes




## Checklist



## Further comments

Translated strings are added only to the English language. Still need to add translated strings to other languages.
